### DISCUSSION TITLE
Mobu 2022?

### DISCUSSION THREAD

**Discussion by readergeneral** [Neill3d/OpenMoBu#10] [General]
thanks for the great job and effort
any "face cap" streaming for mobu 2022?
or, can you create an installer (exe) for mobu 2022...

thanks in advance

**Comment by Neill3d** [comment]
Please, Try out this pre release installation package - https://github.com/Neill3d/OpenMoBu/releases/tag/2021

**Reply by MYAzab** [reply]
Hi Neill,
Thanks for the great effort,..
i have tried to connect and stream my prepared model (a face with blendshapes only) but no luck, 

- The OSC Device working in the yellow mode (no green/ green flashes).
- I have tried to connect using the port (9000).. 
- I have used the IP (192.168.1.81)/ also (*.1.39)..
- The blendshapes are all named and connected using the rotation constraints (node-based connecter), and all was active..

* I bought the facecap app just to make sure that the problem is not related to the limited version..

Note: i even tried to use the actor face and the character face. also no luck..
i have followed the youtube video steps and repeated the steps many times (but no luck):
https://www.youtube.com/watch?v=PFMy9Q0beC0&t=381s&ab_channel=luisgrifu

it seems that there is no streaming / live data..

and advices or something?
and again.. thanks you for every moment you spend on this..

**Reply by Neill3d** [reply]
just tried with facecap app, it works. Quite important to have your phone and your computer in the same local network, then first of all run a mobu device and check your computer IP address, you can do that from command line console application, just type "ipconfig"
When mobu device is running, you can start your facecap and enter your computer IP address, connect.

**Reply by MYAzab** [reply]
yes it works, and that was convincing,.. thanks a lot.
kindly, i would love to buy you a coffee, any way / link ?

**Reply by Neill3d** [reply]
if you would like to buy me a coffee ), here is my paypal - paypal.me/neill3d
